pkg/services/ngalert/api/api_ruler.go

```diff
@@ -27,6 +28,8 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
+const NAMESPACE_SEPARATOR = "/"
```

Comment:
If we only use forward slashes - perhaps we could use path.Join vs strings.Join?

Reply:
on second thought I think that this is better (no change for existing flat folders) so I have followed your suggestion